modules/custom/az_event/config/install/core.entity_view_display.node.az_event.az_card.yml

```
@@ -0,0 +1,308 @@
+      weight: 21
+      format_type: html_element
+      region: hidden
+      format_settings:
+        id: ''
+        classes: col-md-6
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+      label: 'column 1'
+    group_col_2:
+      children:
+        - field_az_location
+      parent_name: group_row_1
+      weight: 22
+      format_type: html_element
+      region: hidden
+      format_settings:
+        id: ''
+        classes: col-md-6
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+      label: 'column 2'
+    group_col_3:
+      children: {  }
+      parent_name: group_row_2
+      weight: 21
+      format_type: html_element
+      region: hidden
+      format_settings:
+        id: ''
+        classes: col-md-6
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+      label: 'column 3'
+    group_col_4:
+      children:
+        - field_az_attachments
+      parent_name: group_row_2
+      weight: 22
+      format_type: html_element
+      region: hidden
+      format_settings:
+        id: ''
+        classes: col-md-6
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+      label: 'column 4'
+    group_card_clickable:
+      children:
+        - group_link
+      parent_name: ''
+      weight: 0
+      format_type: html_element
+      region: content
+      format_settings:
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+        id: ''
+        classes: 'card card-borderless card-clickable bg-gray-100 mb-0 mt-0 mb-4 p-0'
+      label: 'Card Clickable'
+    group_link:
+      children:
+        - group_date
+        - group_heading
+        - group_summary
+      parent_name: group_card_clickable
+      weight: 1
+      format_type: link
+      region: content
+      format_settings:
+        target: entity
+        custom_uri: ''
+        target_attribute: default
+        id: ''
+        classes: 'card-body p-4'
+      label: Link
+    group_date:
+      children:
+        - field_az_event_date
+      parent_name: group_link
+      weight: 5
+      format_type: html_element
+      region: content
+      format_settings:
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+        id: ''
+        classes: 'text-muted font-weight-bold small mt-0 mb-0'
```

```suggestion
        classes: 'text-muted font-weight-bold small'
```
(margin classes aren't needed here)